Makefile

```diff
@@ -37,6 +37,7 @@ $(RUNTIME_TOOLS_LINK):
 	ln -sf $(CURDIR) $(RUNTIME_TOOLS_LINK)
 
 localvalidation:
+	go build -tags "$(BUILDTAGS)" -o runtimetest ./cmd/runtimetest
```

Comment:
We can create a separate target for runtimetest and make localvalidation depend on it.